css-scroll-snap-2/Overview.bs

```diff
@@ -601,3 +603,46 @@ Physical Longhands for 'scroll-start-target' {#scroll-start-target-longhands-phy
 	</pre>
 
 	...
+
+Appendix B: Event Handlers {#event-handlers}
+============================================================
+
+ISSUE: This section should be moved to the HTML event handler
+<a href="https://html.spec.whatwg.org/#event-handlers-on-elements,-document-objects,-and-window-objects">specification</a>.
+
+Event handlers on elements, Document objects and Window objects {#event-handlers-on-elements-document-and-window-objects}
+------------------------------------------------------------------------------------------------------
+
+	The following are the <a>event handlers</a> (and their corresponding <a>event handler event types</a>)
```

Comment:
nit: Maybe say additional event handlers? Otherwise you could read this as a replacement